### PR TITLE
fixing some psr-2 issues

### PR DESCRIPTION
**Pull request by teiling88**

### CODE REVIEW

**Comment by following5** [comment]
Test ok.

**Comment by following5** [comment]
Sorry, I tested master branch instead of your change. Test not ok. ;)

File: htdocs/mylists.php [hunk 1]
```diff
@@ -5,197 +5,187 @@
...
-    $edit_list = false;
-    $name_error = false;
-
-    // open a 'create new list' form
-    if (isset($_REQUEST['new']))
-    {
+require __DIR__ . 'lib2/web.inc.php';
+require_once __DIR__ . 'lib2/logic/cachelist.class.php';
+require_once __DIR__ . 'lib2/edithelper.inc.php';
+
```
Comment (following5) [inline htdocs/mylists.php]: slashes are missing before 'lib2/...'

Comment (following5) [inline htdocs/mylists.php]: and the cachelist.class.php include is no longer needed

**Comment by following5** [comment]
Huh, the merge comment got into the commit comment.

"Everything except for the require paths is fine. As this PR blocks http://redmine.opencaching.de/issues/969, I will just merge it and then correct the paths by a separate commit to master."

**Comment by teiling88** [comment]
seriously? you clicket 2 times on a merge button accidentally?

**Comment by following5** [comment]
I clicked intentionelly on the merge button, but thought that my comment - the explanation for the merge - would [also] go into the PR discussion.

**Comment by following5** [comment]
1931c6f: There is no `__DIR__` in any require statement in the htdocs directory, because it's not needed. So why make an exception for mylists.php?

**Pull request merged**